File: Tareas/Unidad2/a2253332351_practica02/12_ConversionCelsiusFahrenheit.java
import java.util.Scanner;

public class ConversionCelsiusFahrenheit {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double celsius, fahrenheit;

          System.out.println("Ingrese la temperatura en grados Celsius: ");
          celsius = entrada.nextDouble();

          fahrenheit = (celsius * 9 / 5) + 32;

        System.out.println("La temperatura en grados Fahrenheit es: " +
fahrenheit);
    }
}

File: Tareas/Unidad2/a2253332351_practica02/12_ConversionCelsiusFahrenheit.cpp
#include <iostream>
using namespace std;

int main() {
    double celsius;
    double fahrenheit;

    cout << "Ingrese la temperatura en grados Celsius: ";
    cin >> celsius;

    fahrenheit = (celsius * 9 / 5) + 32;

    cout << "La temperatura en grados Fahrenheit es: " << fahrenheit << endl;

    return 0;
}
